### ISSUE TITLE
508-defect-3 [SCREENREADER]: VAOS - "State" in clinic location does not announce the state name

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#18678]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Noah Gelman

## User Story or Problem Statement

"State" in clinic location does not announce the state name, just the sounds of the state initials

## Details

https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-appointment/va-facility- (in locations list and in "Why isn't my facility listed?" content)

When a full address is displayed, a screen reader knows to read the whole state name, but when it's just a city/state, it does not.

`Seattle, WA` announces as "Seattle Wah"

## Acceptance Criteria

- [ ] Add aria-label to state initials so they can be announced correctly Seattle, `<span aria-label="Washington">WA</span>`

## WCAG or Vendor Guidance (optional)

https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-located.html
